src/braket/default_simulator/openqasm/parser/braket_pragmas.py

```diff
@@ -213,7 +213,7 @@ def visitProbabilities(self, ctx: BraketPragmasParser.ProbabilitiesContext):
         return [float(prob.symbol.text) for prob in ctx.children[::2]]
 
 
-def parse_braket_pragma(pragma_body: str, qubit_table: "QubitTable"):
+def parse_braket_pragma(pragma_body: str, pragma_node_visitor=None):
```

Comment:
What's the motivation for this change? I don't think program context should need to import and instantiate a pragma node visitor, rather than just passing the qubit table

Reply:
When I made the pragma node visitor abstract I changed it to accept pragma node visitor.
I felt like maybe in the future if someone wants to modify the behavior like this.

I will change it back to accept qubit table. I don't think anyone would change the behavior of pragma node visitor from program context